docs/linux-server-setup.md

```diff
@@ -76,7 +109,31 @@ They follow this structure:
 Variablename = "MySQLIP;Port;Username;Password;database"  
 ``` 
 
-<br>
+The following steps must be verified:
+
+- The hostname (127.0.0.1) can stay the same if AzerothCore is being installed on the same computer that you run WoW on.
+  If not, follow the instruction in [Realmlist Table](realmlist).
+
+- The port (3306) is the standard configured value. If you changed the default port in your MySQL settings, you must change it accordingly.
+  The username and password can be variable. You can choose to either: 
+
+    - use default acore / acore username and password pair.
+
+    - create a unique login within a User Manager within your preferred database management tool (commonly identified by an icon that looks like a person or people) and give it the necessary permissions (SELECT, INSERT, UPDATE, DELETE permissions are sufficient, and is much safer).
+
+### Updating DataDir
+
+1. In your worldserver.conf file locate the **DataDir** option.
+
+1. Edit it to the path of your folder. i.e **$HOME/azerothcore/data/**
```

Comment:
worldserver.conf has a generic example in the config. continue using our example folder here

```suggestion
1. Edit it to the path of your folder. e.g. **$AC_CODE_DIR/build/data**
```